### PR TITLE
Fixing implicit conversion warnings.

### PR DESCRIPTION
**Pull request by jacobsologub**

### CODE REVIEW

File: src/WebSocketProtocol.h [hunk 1]
```diff
@@ -148,7 +148,7 @@ class WIN32_EXPORT WebSocketProtocol {
         } else {
             wState->state.spillLength = 0;
             wState->state.wantsHead = false;
-            wState->remainingBytes = payLength - length + MESSAGE_HEADER;
+            wState->remainingBytes = (unsigned int) payLength - length + MESSAGE_HEADER;
```
Comment (alexhultman) [inline src/WebSocketProtocol.h]: This causes a bug

Comment (alexhultman) [reply src/WebSocketProtocol.h]: You need parenthesis here otherwise you mess up the logic

File: src/Networking.h [hunk 2]
```diff
@@ -216,7 +216,7 @@ struct WIN32_EXPORT NodeData {
     static void asyncCallback(Async *async);
 
     static int getMemoryBlockIndex(size_t length) {
-        return (length >> 4) + bool(length & 15);
+        return (int) (length >> 4) + bool(length & 15);
```
Comment (alexhultman) [inline src/Networking.h]: Same here, you change the logic.

**Comment by jacobsologub** [comment]
Fixed

**Pull request merged**